docs/user-guide/meetings/dynamic-meetings/README.md

```diff
@@ -142,6 +143,18 @@ Once a meeting has been closed, it can no longer be edited. Project members with
 
 ![Re-open a meeting in OpenProject](openproject_dynmic_meetings_reopen_meeting.png)
 
+## Copy a meeting
+
+You can copy an existing meeting. This is useful if you have recurring meetings. To copy a meeting, click on the three dots in the top right corner and select **Copy**. 
+
+![Copy a dynamic meeting in OpenProject](openproject_dynamic_meetings_copy_meeting.png)
+
+A screen will open, which will allow you adjust the name, time, location and further details of the copied meeting. You also have an option of copying the agenda. Don't forget to **Save** the copied meeting.
```

Comment:
You could add a note here that by default, the new meeting's date will be advanced by one week

Reply:
thank you, excellent point. I will add it.